alf/environments/utils.py

```diff
@@ -185,7 +185,7 @@ def create_environment(env_name='CartPole-v0',
         _get_wrapped_fn(env_load_fn), 'no_thread_env', False)
 
     if nonparallel:
-        num_parallel_environments = 1
+        assert num_parallel_environments == 1, "nonparallel is True"
```

Comment:
update the doc-string

Reply:
Done, and added extra assert on batch_size_per_env.  PTAL.

Reply:
Where is the extra assert on batch_size_per_env?

Reply:
Ah, pushed to the wrong remote..  Now it's Ok.